tests/backup/backup_partial_test.go

```diff
@@ -991,3 +993,240 @@ var _ = Describe("{BackupStateTransitionForScheduledBackups}", Label(TestCaseLab
 		CleanupCloudSettingsAndClusters(backupLocationMap, cloudAccountName, cloudAccountUid, ctx)
 	})
 })
+
+// This testcase verifies the partial backup success when few Px volumes backups failed while taking backup to Azure Global Location when env variable is set to non-global location.
+var _ = Describe("{PartialBackupSuccessWithAzureEndpoint}", func() {
+
+	var (
+		backupNames                []string
+		restoreNames               []string
+		scheduledAppContexts       []*scheduler.Context
+		sourceClusterUid           string
+		cloudCredName              string
+		cloudCredUID               string
+		backupLocationUID          string
+		backupLocationName         string
+		backupLocationMap          = make(map[string]string)
+		labelSelectors             = make(map[string]string)
+		provisionerVscMap          = make(map[string]string)
+		AzureEnvironmentEnvVar     = make(map[string]string)
+		namespaceMapping           = make(map[string]string)
+		AzureEnvironmentEnvVarName string
+		provider                   string
+		failedVolumes              []*corev1.PersistentVolumeClaim
+		backedUpNamespaces         []string
+		pxProvisioners             []string
+	)
+
+	JustBeforeEach(func() {
+		StartPxBackupTorpedoTest("PartialBackupSuccessWithAzureEndpoint", "verifies the partial backup success when few Px volumes backups failed while taking backup to Azure Global Location when env variable is set to non-global location", nil, 299236, Ak, Q2FY25)
+		// This testcase is specific to Azure provider with Px Volumes
+		provider = "azure"
+		numOfNamespace := 1
+		AzureEnvironmentEnvVarName = "AZURE_ENVIRONMENT"
+		pxProvisioners = []string{"kubernetes.io/portworx-volume", "pxd.portworx.com"}
+		log.InfoD("scheduling applications")
+		scheduledAppContexts = make([]*scheduler.Context, 0)
+		appList := Inst().AppList
+		defer func() {
+			Inst().AppList = appList
+		}()
+		var err error
+		Inst().AppList = []string{"pxb-singleapp-multivol"}
```

Comment:
Can we use `GetApplicationSpecForFeature` here?

Reply:
We need an app that has multiple volumes and data, as KDMP backup is involved.

Reply:
pxb-singleapp-multivol can we have a better naming to figure out which application is being uesd here

Reply:
Thanks for suggestion @sn-px , I'll take up this in my next PR where i'll be adding spec for different  providers as well.